Gemfile

```diff
@@ -5,8 +5,10 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-gem 'spree', github: 'spree/spree', branch: 'master'
-gem 'spree_auth_devise', github: 'spree/spree_auth_devise', branch: 'master'
+gem 'spree', github: 'spree/spree', branch: 'main'
+gem 'spree_auth_devise', github: 'spree/spree_auth_devise', branch: 'main'
 gem 'rails-controller-testing'
+gem 'byebug'
```

Comment:
I'm not sure if this one should be in the Gemfile (even if it's for development) - this is more of a utility, rather than something that's required for development